en_US/messages.json

```
@@ -134,5 +134,9 @@
           "example" : "100"
         }
       }
+    },
+    "sslIncompatibility": {
+      "message": "Translations doesn't work on non-secure (https) pages.",
```

```suggestion
      "message": "Translations doesn’t work on non-secure (HTTPS) pages.",
```

---

```suggestion
      "message": "Translations only works on secure (HTTPS) pages.",
```

As is, it the parenthetical bit seems to be clarifying the "secure" part of "non-secure", which is a bit confusing and likely not to translate very well.

---

Good point. Also, double negations are evil.